### ISSUE TITLE
nilearn.datasets.fetch_icbm152_2009 docstring isn't great

### ISSUE THREAD

**Issue by GaelVaroquaux** [nilearn/nilearn#1166]
The description of the return argument in nilearn.datasets.fetch_icbm152_2009 is hard to read.

Also, we should add a seealso block to point to nilearn.datasets.load_mni152_template and fetch_icbm152_brain_gm_mask

**Comment by KamalakerDadi** [comment]
> The description of the return argument in nilearn.datasets.fetch_icbm152_2009 is hard to read.

I always ```print(description)``` which gives me clear picture of the description. Do you mean it should be easy to read without ```print()``` ?

**Comment by GaelVaroquaux** [comment]
> I always print(description) which gives me clear picture of the description. Do
> you mean it should be easy to read without print() ?

I mean in the docstring.

**Comment by KamalakerDadi** [comment]
Any reference you can point for example to look at ?

**Comment by bthirion** [comment]
@KamalakerDadi are you willing to do that ?

**Comment by KamalakerDadi** [comment]
Sure let me reopen the PR.

Could you summarize what needs to be done from your understanding? @bthirion
